packages/compass-crud/src/components/insert-document.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { css, DocumentList, spacing } from '@mongodb-js/compass-components';
+import type HadronDocumentType from 'hadron-document';
+
+const insertDocumentStyles = css({
+  // We give it a good amount of spacing for dropdown menus.
+  paddingBottom: spacing[7],
```

Comment:
Do we have a ticket to properly fix it or are we not planning to?

Reply:
Created one https://jira.mongodb.org/browse/COMPASS-6271 it's a bit more involved as we'll want to add a scroll handler I think. Updating the comment to include a link to the ticket 👍